### DISCUSSION TITLE
assign one reviewer as blocking reviewer

### DISCUSSION THREAD

**Discussion by mingmxu** [community/community#102489] [Pull Requests] [answered]
### Select Topic Area

General

### Body

In many cases, we need to put someone as the blocking reviewer(I don't see the option in PR UI) before a PR is fully approved. Usually they're domain experts, heavy relies on the service, or other scenarios. It's case-by-case, usually the author or another reviewer decides to get the authority before code is shipped silently. 

Adding them to reviewer list currently doesn't work here. Code owner is not a good option, usually the blocking reviewer doesn't own the code, they care a small part of all changes.

**Comment by queenofcorgis** [comment, marked as answer]
Hey there @mingmxu thanks for sharing this feedback and explaining so well your use case, I added the proper label so it can be reviewed  by our team. Cheers!
